libnmstate/nm/applier.py

```diff
@@ -90,7 +92,10 @@ def apply_changes(context, net_state, save_to_disk):
     pending_con_profiles = []
 
     for iface_desired_state in filter(
-        lambda s: s.get(Interface.STATE) != InterfaceState.ABSENT,
+        lambda s: s.get(Interface.STATE) == InterfaceState.UP
+        or (
+            s.get(Interface.STATE) == InterfaceState.DOWN and s.get(IS_DESIRED)
```

Comment:
This clearly will break a lot in CI.

Reply:
Not needed anymore. I've used your suggestions